.vscode/settings.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
add this to the .gitignore we don't want this in the repo it is your vscode settings